deploy_partition.yaml

```diff
@@ -10,9 +10,9 @@
 
     - name: check vagrant version
       fail:
-        msg: "Vagrant version of the host system ({{ lookup('file', playbook_dir + '/.vagrant_version_host_system') }}) does not match vagrant version in the deployment image ({{ result.stdout.split(' ')[1] }})"
+        msg: "Vagrant version of the host system ({{ lookup('file', '{{ playbook_dir }}/.vagrant_version_host_system') }}) does not match vagrant version in the deployment image ({{ result.stdout.split(' ')[1] }})"
```

Comment:
I don't think that Jinja properly replaces this... the old variant should work and looks simpler.